lib/main.rb

```diff
@@ -1,3 +1,6 @@
+require_relative 'app'
+require_relative 'movie'
```

Comment:
hi @najibullahjafari, you did a great job with the `app.rb` file.

I think it would be better to use the `App` class as a junction point for all our other classes: "Book", "Game", "Music", and "Movie" so we could require_relative them all in the class `App` and after require_relative the `App` class in the `main` file